bp_top/test/tb/bp_tethered/Makefile.params

```diff
@@ -16,6 +16,7 @@ export CHECKPOINT_P   ?= 0
 export COSIM_P        ?= 0
 export PRELOAD_MEM_P  ?= 0
 export NBF_CONFIG_P   ?= 1
+export SYNTH_P        ?= 0
```

Comment:
default to SIM_SYNTH_VIVADO_P | SIM_SYNTH_DC_P

Reply:
And rename to NO_BIND_P (there might be cases where we don't want to bind anything, even without it being a synth wrapper)